src/tests/JIT/Regression/JitBlue/Runtime_620/Runtime_620.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Thanks for the test, could you please add a few with unpromoted struct fields?

Reply:
I added struct field versions of the four tests methods.